code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -30,20 +31,23 @@
 /obj/structure/closet/cardboard/open()
 	if(opened || !can_open())
 		return 0
-	if(!egged)
+	var/list/alerted = null
+	if(world.time - egged > SNAKE_SPAM_TICKS)
```

Comment:
this is a very nonstandard way to do a world.time cooldown, the standard way would be;

    if(egged < world.time) //or world.time > egged but that's just style choice
    	stuff
    egged = world.time + SNAKE_SPAM_TICKS